### ISSUE TITLE
Add start and stop parameters to the range.index() ABC method

### ISSUE THREAD

**Issue by fe5a23f9-4d47-49f8-9fb5-d6fbad5d9e38** [python/cpython#72384]
BPO | [28197](https://bugs.python.org/issue28197)
--- | :---
Nosy | @rhettinger, @ssbr, @serhiy-storchaka, @vedgar, @nitishch, @shreyanavigyan
PRs | <li>python/cpython#4378</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2016-09-18.20:50:32.845>
labels = ['interpreter-core', 'easy', 'type-feature', '3.7']
title = 'Add start and stop parameters to the range.index() ABC method'
updated_at = <Date 2021-04-14.18:06:12.283>
user = 'https://github.com/vedgar'
```

bugs.python.org fields:
```python
activity = <Date 2021-04-14.18:06:12.283>
actor = 'shreyanavigyan'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Interpreter Core']
creation = <Date 2016-09-18.20:50:32.845>
creator = 'veky'
dependencies = []
files = []
hgrepos = []
issue_num = 28197
keywords = ['patch', 'easy (C)']
message_count = 12.0
messages = ['276908', '276912', '276930', '276933', '276946', '277156', '277162', '277163', '277179', '277198', '305548', '307833']
nosy_count = 7.0
nosy_names = ['rhettinger', 'Devin Jeanpierre', 'docs@python', 'serhiy.storchaka', 'veky', 'nitishch', 'shreyanavigyan']
pr_nums = ['4378']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue28197'
versions = ['Python 3.7']
```

</p></details>

**Comment by serhiy-storchaka** [comment]
Opened bpo-31942 for the documenting part.

**Comment by iritkatriel** [comment]
The doctoring was corrected in #79029 to remove start and stop. I think this issue can be closed, and if someone wants to suggest it as a feature request, they would write to python ideas first and then create a new issue.
